app/models/volunteer_op.rb

```diff
@@ -3,10 +3,18 @@ class VolunteerOp < ActiveRecord::Base
   validates :title, :description, presence: true
   validates :organisation_id, presence: true, if: "source == 'local'"
   belongs_to :organisation
-
+  
+  geocoded_by :full_address
+  after_validation :geocode, if: :different_address?   
+  after_validation :clear_lat_lng
```

Comment:
Hi @tansaku 
the idea was that if the user leave the different address fields blank the geocoding falls back to the organisation address which happend only if the volounter_ops has no address ...